recipes/platform.delegates/all/conanfile.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class PlatformInterfacesConan(ConanFile):
+    name = "platform.delegates"
+    license = "MIT"
+    homepage = "https://github.com/linksplatform/Delegates"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = """platform.delegates is one of the libraries of the LinksPlatform modular framework, which uses 
+    innovations from the C++17 standard, for easier use delegates/events in csharp style."""
+    topics = ("linksplatform", "cpp17", "delegates", "events", "header-only")
+    settings = "os", "compiler", "build_type", "arch"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _subfolder_sources(self):
```

Comment:
This confused me.... it looks like the usually one

Can we please rename this?

```suggestion
    def _internal_cpp_subfolder(self):
```

Reply:
It probably won't cause any problems

Reply:
For machine, no problem. For humans, it looks ambiguous. Consider Prince's suggestion.